src/universal/conf/private-testnet.conf

```diff
@@ -0,0 +1,74 @@
+# Sample configuration for a custom private testnet.
+mantis {
+  sync {
+    # Whether to enable fast-sync
+    do-fast-sync = false
+
+    # Duration for blacklisting a peer. Blacklisting reason include: invalid response from peer, response time-out, etc.
+    # 0 value is a valid duration and it will disable blacklisting completely (which can be useful when all nodes are
+    # are controlled by a single party, eg. private networks)
+    blacklist-duration = 0
+
+    # Set to false to disable broadcasting the NewBlockHashes message, as its usefulness is debatable,
+    # especially in the context of private networks
+    broadcast-new-block-hashes = false
+  }
+
+  blockchain {
+    # DAO fork configuration (Ethereum HF/Classic split)
+    # https://blog.ethereum.org/2016/07/20/hard-fork-completed/
+    dao = null
+
+    # Custom genesis JSON file
+    # null value indicates using default genesis definition that matches the main network
+    custom-genesis-file = "conf/private-genesis.json"
+  }
+
+  network {
+    discovery {
+      # Listening interface for discovery protocol
+      interface = ${mantis.network.server-address.interface}
+    }
+
+    peer {
+      # Ethereum network identifier:
+      # 1 - mainnet, 2 - morden
+      network-id = 2139
+    }
+
+    rpc {
+      # JSON-RPC mode
+      # Available modes are: http, https
+      # Choosing https requires creating a certificate and setting up 'certificate-keystore-path' and
+      # 'certificate-password-file'
+      # See: https://github.com/input-output-hk/mantis/wiki/Creating-self-signed-certificate-for-using-JSON-RPC-with-HTTPS
+      mode = "http"
+
+      # Listening address of JSON-RPC HTTP/HTTPS endpoint
+      interface = ${mantis.network.server-address.interface}
+
+      # Domains allowed to query RPC endpoint. Use "*" to enable requests from any domain.
+      cors-allowed-origins = []
```

Comment:
I wonder if we should set this to `"*"`

Reply:
Yes, I changed this to match exactly what we do now in the testnet, which is actually the primary motivation for this PR.